include/crypto/evp.h

```diff
@@ -751,6 +754,15 @@ struct evp_pkey_st {
 # define EVP_PKEY_CTX_IS_KEM_OP(ctx) \
     (((ctx)->operation & EVP_PKEY_OP_TYPE_KEM) != 0)
 
+struct evp_skey_st {
+    /* == Common attributes == */
+    CRYPTO_REF_COUNT references;
+    CRYPTO_RWLOCK *lock;
+
+    void *keydata; /* Alg-specific key data */
+    EVP_SKEYMGMT *skeymgmt; /* Import, export, manage */
+}; /* EVP_SKEY */
```

Comment:
Why is this in a public header?

Reply:
I guess this is not the public header.

Reply:
I think the public headers are in `include/openssl/`